src/test/java/com/mercadopago/client/order/OrderClientIT.java

```diff
@@ -270,7 +271,7 @@ public void totalRefundOrderSuccess() {
 
       Order order = client.create(orderCreateRequest);
       Thread.sleep(3000);
-      Order refundedOrder = client.refund(order.getId());
+      OrderRefund refundedOrder = client.refund(order.getId());
```

Comment:
[req] O endpoint de refund retorna uma resposta do tipo Order, é que no Swagger o pessoal deu uma encurtada no exemplo, parece uma resposta diferente mas é uma Order.
Como tratamos todos os campos como opcionais dentro das SDKs, não tem problema usar Order aqui.
```suggestion
      Order refundedOrder = client.refund(order.getId());
```